moby_simulator.py

```diff
@@ -89,8 +90,8 @@ def main():
 		for current_hour in list(range(0,24)):
 			network_state_new = defaultdict(set)
 			current_data_file = DATA_FILE_PREFIX + str(city_number) + "/" + str(current_day) + "_" + str(current_hour) + DATA_FILE_FORMAT
-			current_data_file_threshold = DATA_FILE_PREFIX + str(city_number) + "_" + str(threshold) + "/" + str(current_day) + "_" + str(current_hour) + DATA_FILE_FORMAT
-
+			#current_data_file_threshold = DATA_FILE_PREFIX + str(city_number) + "_" + str(threshold) + "/" + str(current_day) + "_" + str(current_hour) + DATA_FILE_FORMAT
+                        current_data_file_threshold = DATA_FILE_PREFIX + str(city_number) + "_" + str(threshold) + "/" + str(start_day) + "/" + str(numdays) + "/" + str(current_day) + "_" + str(current_hour) + DATA_FILE_FORMAT
```

Comment:
Again, just replace the line you've changed. No point keeping the older one as a comment.